### PR TITLE
Feat/comment on ama submissions

### PR DESCRIPTION
**Pull request by SarrankanPathmanathan**
This adds the ability to add comments to an AMA

### CODE REVIEW

File: src/main/java/ama/post/CommentPost.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public CommentPost(User op, Post context, String text) {
     }
 
     public void setContext(Post post) {
-        this.context = context;
+        this.context = post;
```
Comment (RyanBillard) [inline src/main/java/ama/post/CommentPost.java]: 💥

File: src/main/java/ama/post/CommentPostRepository.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
  * Created by stephanernst on 3/20/2017.
  */
 @Transactional
-public interface CommentPostRepository extends PostRepository {
+public interface CommentPostRepository extends PostRepository<CommentPost> {
```
Comment (RyanBillard) [inline src/main/java/ama/post/CommentPostRepository.java]: What does this accomplish?

Comment (stephanernst) [inline src/main/java/ama/post/CommentPostRepository.java]: I don't even remember making this change to be honest.

Comment (SarrankanPathmanathan) [inline src/main/java/ama/post/CommentPostRepository.java]: It was done in SubmissionPostRespository and I assumed it had to be for this repository as well. It made retrieving the saved comment work.

File: src/main/resources/templates/pageNotFound.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8"/>
```
Comment (stephanernst) [inline src/main/resources/templates/pageNotFound.html]: Nice.

**Pull request merged**